### DISCUSSION TITLE
Community Meeting - August 9, 2022 9-10AM PDT

### DISCUSSION THREAD

**Discussion by theadactyl** [openxla/community#5] [Announcements]
Hi folks! We've posted a [proposed agenda](https://github.com/openxla/community/wiki/20220809---OpenXLA-Community-Meeting) & meeting details for our first OpenXLA Community Meeting. 

Please let us know here if there are any additional topics you'd like to discuss.

**Comment by bhack** [comment]
As we are moving this from the TF repository how we are going to name the bridge components that will still be maintained in the TF repository?

I am asking this also related to the TF forum as we have an `XLA` tag. As we are having a new vendor neutral discussion space here on GitHub Discussion and with Discord I want to understand how we are going to naming the bridge component in the TF forum and in GitHub TF label.
I think that still generically call it XLA (or OpenXLA) in TF and here it could be really confusing to route the users, tickets, threads etc. so I hope that when could find a new specific name in TF than the generic XLA or OpenXLA.

Probably It is more a TF specific issue but IMHO still valid here as we are generating this repository from the official TF source one.

**Reply by joker-eph** [reply]
The bridge is currently named TF/XLA or TF2XLA, it has always been and I don't think we should change this.
How TF will manages its labels, issues, categories (whether to rename or keep "XLA") is TBD, and mostly not in scope of "OpenXLA".

**Comment by etherzhhb** [comment]
I remember currently XLA is built with Bazel, do we plan to move away from Bazel, and use something like CMake instead?

**Reply by joker-eph** [reply]
It's an excellent topic, I filed an issue to discuss/track it further: https://github.com/openxla/xla/issues/1 ; feel free to chime-in there!

**Comment by theadactyl** [comment]
Hi folks -- the meeting slides, minutes, and attendee list are available here: https://github.com/openxla/community/tree/main/meetings/20220809-community-meeting 

Please feel free to add yourself (or remove) from the attendee list via PR! We are also compiling the Q&A and will add to the meeting folder when it's ready.
